src/lib/components/OrganizationCard/OrganizationCard.svelte

```diff
@@ -0,0 +1,98 @@
+<script lang="ts">
+	import { typography } from '../../../styles/tailwind/typography';
+	import { cn } from '../../utils/cn';
+	import arrowRight from '../../assets/icons/arrow-right.svg';
+	import arrowCircleRight from '../../assets/icons/arrow-circle-right.svg';
+	import sapanisit from '../../assets/images/sapanisit.png';
+	import smo from '../../assets/images/smo.png';
+	import sgcu from '../../assets/images/sgcu.png';
+
+	const organizations = [
+		{
+			imageURL: smo,
+			orgName: 'สโมสรนิสิต \n จุฬาลงกรณ์มหาวิทยาลัย',
+			linkHref: 'https://www.google.com'
+		},
+		{
+			imageURL: sgcu,
+			orgName: 'องค์การบริหารสโมสรนิสิต จุฬาลงกรณ์มหาวิทยาลัย (อบจ.)',
+			linkHref: 'https://www.chula.ac.th/academics/academic-units/sgcu/'
+		},
+		{
+			imageURL: sapanisit,
+			orgName: 'สภานิสิต \n จุฬาลงกรณ์มหาวิทยาลัย',
+			linkHref:
+				'https://www.sa.chula.ac.th/%E0%B8%AA%E0%B8%A0%E0%B8%B2%E0%B8%99%E0%B8%B4%E0%B8%AA%E0%B8%B4%E0%B8%95/'
+		}
```

Comment:
can you move this part of code into `src/lib/constant` and add a file `organizations.ts` and import this variables to this part of code instead ?